### ISSUE TITLE
[Bug] `DrawingView` shows no output without setting `LineColor`

### ISSUE THREAD

**Issue by jfversluis** [xamarin/XamarinCommunityToolkit#1452]
### Description

Because the default value of `LineColor` is `Color.Default` no line is shown when you don't set it explicitly. This is not a great first-time use experience. I would set it to `Color.Black` by default so that at least something shows when using this for the first time.

### Steps to Reproduce

1. Implement a `DrawingView` like below

```xaml
<xct:DrawingView BackgroundColor="LightGray" HeightRequest="200" Points="{Binding DrawPoints}" DrawingCompletedCommand="{Binding DrawCompleted}" />
```

2. Draw on the canvas
3. Notice how nothing appears

### Basic Information

- Version with issue: Current nightly
- Last known good version: N/A
- IDE: VSMac
- Platform Target Frameworks: 
  - iOS:  14.5
  - Android: Not tested, but since this happens in shared code I expect this to happen here too
  - UWP:   

### Workaround

Set `LineColor` property to something explicitly

**Comment by jfversluis** [comment]
@VladislavAntonyuk FYI

**Comment by VladislavAntonyuk** [comment]
Fixed. Set LineColor to Black

**Comment by jfversluis** [comment]
Fixed in #1459
